### ISSUE TITLE
Query does not work with windows embedded Ubuntu

### ISSUE THREAD

**Issue by Bluscream** [TeaSpeak/TeaSpeak#8]
## System Information
***Operating System***: Windows 10 Integrated Ubuntu shell
***Operating System Type*** (x86|x64): x64
***TeaSpeak Version***: 1.0.8

## Issue
```
> Allocate query connection
[12:42:25] [INFO] Got new client!
[12:42:25] [INFO] Having currently highest cldbid = 1
[12:42:25] [DEBUG] Read db value cldbid - 2
[12:42:25] [DEBUG] Read db value lastConnect - 0
[12:42:25] [DEBUG] Unknown row name 'lastConnect'
[12:42:25] [DEBUG] Read db value connections - 0
[12:42:25] [DEBUG] Load data for server -> 0 dbid = 2
[12:42:25] [DEBUG] [SQL] UPDATE `clients` SET `lastName` = :value WHERE `serverId` = :sid AND `cldbid` = :cldbid - 0 - ServerQuery#127.0.0.1/3899 - 2
[12:42:25] [INFO] Got client fom db. ClientID: 2
[12:42:25] [INFO] [Query] Got new client from 127.0.0.1:3899
[12:42:25] [ERROR] Query invalid read. Code: -1 errno: 22 msg: Invalid argument
> Delete query connection
```
  
I also tried Windows's Telnet client but it kicks me out as soon as i start to write:
```
TS3
   Welcome to the TeaSpeak ServerQuery interface, type "help" for a list of commands and "help <command>" for information on a specific command.
       v

Connection to host lost.

Press any key to continue...
```

Same thing with KiTTY
![](https://i.imgur.com/7iyXNbC.png)

**Comment by WolverinDEV** [comment]
Hmm i guess that a side effect of the environment where you running teaspeak.
Do you also tested it under Linux?

As a little question:
Which setup do you use?

**Comment by Bluscream** [comment]
What do you mean by Setup? :o

**Comment by WolverinDEV** [comment]
How you executed the Linux binary under windows

**Comment by Bluscream** [comment]
https://docs.microsoft.com/en-us/windows/wsl/install-win10
https://www.microsoft.com/en-us/store/p/ubuntu/9nblggh4msv6

**Comment by WolverinDEV** [comment]
Fixed in 1.0.11-Beta
